### ISSUE TITLE
make data accessible via zenodo.org?

### ISSUE THREAD

**Issue by kkmann** [CSSEGISandData/COVID-19#910]
Dear all,

thanks for putting this amazing resource out!

Did you consider making the data available via https://zenodo.org? Zenodo allows DOI-versioned releases of scientific data sets and would make it easier to reference the exact version of the data used for downstream analyses. 

Best wishes,

Kevin

**Comment by klartext** [comment]
Before or after cleaning the data?!

**Comment by klartext** [comment]
Is this a scientific dataset at all?
What does data make a scientifc dataset?

**Comment by kkmann** [comment]
Good point, zenodo has a very open definition though and given the resonance to your work, I think it qualifies as scientific ;) 

Well, maybe both? I think the default limit per repo is around 50gb. Also, you can release via GitHub releases if you just want to mirror the repo there. My main point is that having daily DOIs would be handy instead of linking to the gh repo and quoting the commit hash, might confuse people who are less tech savvy...
